tslint.json

```diff
@@ -68,7 +68,6 @@
     "lifecycle-hook-interface": true,
     "require-breaking-change-version": true,
     "no-nested-ternary": true,
-    "prefer-const-enum": true,
```

Comment:
Looks like this rule would need to be deleted (see file above) / or we change it to enforce **no** const enum?

Reply:
Done!